kernel/src/actions/deletion_vector.rs

```diff
@@ -167,6 +167,8 @@ impl DeletionVectorDescriptor {
         }
     }
 
+    /// Materialize the row indexes of the deletion vector as a Vec<u64> in which each element
```

Comment:
Generally I don't love making any "real" changes in these kinds of PRs. For now it's probably fine, but we should get in the habit of making these first

Reply:
Ah yea agree probably not the best spot for it - just noticed we missed some documentation earlier..

Reply:
Agreed with Nick, but I appreciate you plugging the hole here